resource-managers/mesos/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackendSuite.scala

```diff
@@ -587,6 +587,30 @@ class MesosCoarseGrainedSchedulerBackendSuite extends SparkFunSuite
     assert(networkInfos.get(0).getLabels.getLabels(1).getValue == "val2")
   }
 
+  test("scheduler backend doesn't set '--hostname' for executor when virtual network is enabled") {
```

Comment:
nit. Could you add `SPARK-28778` at the test case since this is a bug fix PR?
```scala
- test("scheduler backend doesn't set '--hostname' for executor when virtual network is enabled") {
+ test("SPARK-28778 scheduler backend doesn't set '--hostname' for executor when virtual network is enabled") {
```

Reply:
Thanks, @dongjoon-hyun. Added Jira ticket reference